workers/grouper/src/index.ts

```diff
@@ -249,17 +249,19 @@ export default class GrouperWorker extends Worker {
     const lastUniqueEvents = await this.findLastEvents(projectId, eventsCountToCompare);
```

Comment:
Probably, we can find event by title using mongo. We dont need to get 60 events for that.